Bottom: 0c7d9ceab0d5df15acf756ee788c788d99747f7d
Top:    ca4ae90d6b4e7312717901f00c11daf8790d0b7a
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-07-21 22:29:48 +0800

Refresh of vhost_net-honor-exported-skb

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index b51a7df..82ec6f7 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -631,7 +631,7 @@ static int vhost_net_rx_peek_head_len(struct vhost_net *net, struct sock *sk)
 		endtime = busy_clock() + vq->busyloop_timeout;
 
 		while (vhost_can_busy_poll(&net->dev, endtime) &&
-		       !sk_has_rx_data(sk) &&
+		       !sk_has_rx_data(rvq, sk) &&
 		       vhost_vq_avail_empty(&net->dev, vq))
 			cpu_relax();
